bundles/org.openhab.binding.luxtronikheatpump/src/main/java/org/openhab/binding/luxtronikheatpump/internal/enums/HeatpumpChannel.java

```diff
@@ -0,0 +1,968 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.luxtronikheatpump.internal.enums;
+
+import javax.measure.Unit;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.core.items.Item;
+import org.openhab.core.library.items.DateTimeItem;
+import org.openhab.core.library.items.NumberItem;
+import org.openhab.core.library.items.SwitchItem;
+import org.openhab.core.library.unit.SIUnits;
+import org.openhab.core.library.unit.Units;
+
+/**
+ * Represents all valid channels which could be processed by this binding
+ *
+ * @author Stefan Giehl - Initial contribution
+ */
+@NonNullByDefault
+public enum HeatpumpChannel {
+
+    // The channel ids are currently based on the variable names used in the heat pumps internal JAVA applet
+    // for possible values see https://www.loxwiki.eu/display/LOX/Java+Webinterface
+    // or https://github.com/Bouni/Home-Assistant-Luxtronik/blob/master/data.txt
+
+    // EN: Flow temperature heating circuit
+    // DE: Vorlauftemperatur Heizkreis
```

Comment:
```suggestion
    /**
     * Flow temperature heating circuit (original: Vorlauftemperatur Heizkreis)
     */
```

Please format these descriptions as Javadoc. This way the IDE's documentation framework can work.